docs/map_state/topics.md

```diff
@@ -0,0 +1,25 @@
+Atlas pre-organizes your data into topics informed by the latent contents of your embeddings. Visually, these are represented by regions of homogenous
+color on an Atlas map. 
+
+You can access and operate on topics programmatically by using the `topics` attribute
+of an AtlasMap.
+
+
+```python
+from nomic import AtlasProject
+
+project = AtlasProject(name='My Project')
+
+map = project.maps[0]
+
+map.topics.df
```

Comment:
nit: same thing here, can we either show an example or explain what the dataframe looks like (e.g. what columns)

Reply:
Resolved by adding examples containing the dataframe structure